### PR TITLE
Staged Springlock

### PR DESCRIPTION
**Pull request by nicholast013**
unknown

## About The Pull Request

This PR was about adding Staged Springlock

## Why It's Good For The Game

Old springlock boring its just snap and killed you, how about staged springlock! Look as someone scream while slowly modsuit exoskeleton insert into their body

## Changelog

:cl:
add: Springlock staged and modsuit cant be remove once springlocked
/:cl:

### CODE REVIEW

**Comment by Mothblocks** [comment]
you accidentally fucked up the changelogs and stuff

**Comment by nicholast013** [comment]
> you accidentally fucked up the changelogs and stuff

yeah

**Comment by cacogen** [comment]
can we get a vid?

**Comment by Wallemations** [comment]
Yea I'd love to get a vid of this

File: code/modules/mod/modules/modules_maint.dm [hunk 1]
```diff
@@ -41,21 +41,53 @@
...
+	playsound(mod.wearer, 'sound/effects/splat.ogg', 50, TRUE, frequency = 0.5)
+	mod.wearer.apply_damage(damage = 50, damagetype = BRUTE, def_zone = BODY_ZONE_CHEST, forced = FALSE, spread_damage = FALSE, sharpness = WOUND_BLUNT)
+	sleep(35)
+	mod.wearer.apply_damage(damage = 45, damagetype = BRUTE, def_zone = BODY_ZONE_HEAD , forced = TRUE, spread_damage = TRUE, sharpness = WOUND_BLUNT)
+	playsound(mod.wearer, 'sound/effects/snap.ogg', 75, TRUE, frequency = 0.5)
+	playsound(mod.wearer, 'sound/effects/splat.ogg', 50, TRUE, frequency = 0.5)
+	mod.wearer.bleed(8)
 	if(!HAS_TRAIT(mod.wearer, TRAIT_NODEATH))
-		mod.wearer.death() //just in case, for some reason, they're still alive
+		mod.wearer.death()
```
Comment (MrMelbert) [inline code/modules/mod/modules/modules_maint.dm:87]: Same here, any reason in particular why you nuked the comment?

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: code/modules/mod/modules/modules_maint.dm (hunk 1 above)
Comment (LemonInTheDark) [inline code/modules/mod/modules/modules_maint.dm:87]: Much of what you're doing here is the same for each effect, right? Is there any way to reduce the copypasta?
I'm also really not a fan of the sleeps, timers are more controllable, and even if this is unlikely to ever happen more then once or twice a round, a much better pattern

Comment (Mothblocks) [reply code/modules/mod/modules/modules_maint.dm:87]: stoplag wouldve worked from waht i know

**Comment by interestingusernam3** [comment]
i agree

**Pull request closed without merging**